index.html

```diff
@@ -0,0 +1,143 @@
+            </p>
+        </div>
+        <h2>Latest FIFA News</h2>
+        <div class="headlin-description">
+            <p>
+                FIFA is working on a wide range of topics from improving the women's game and developing football around the world to ensuring better football governance structures.
+            </p>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="title-program">
+            <h3>Main program</h3>
+        </div>
+        <div class="cards">
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/champions.png" alt="champions">
+                </div>
+                <div class="title-card">
+                    <h4>champions</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/laliga.png" alt="laliga">
+                </div>
+                <div class="title-card">
+                    <h4>La liga</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/EnglishPremierLeague.webp" alt="premier">
+                </div>
+                <div class="title-card">
+                    <h4>premier league</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/serie.jpg" alt="serie">
+                </div>
+                <div class="title-card">
+                    <h4>serie A</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="featured" id="Featured">
```

Comment:
- [ ] Good job so far by making your speaker section load dynamically :clap: But we have an issue with the design :cry: Please there is a difference between your design and the original design make sure that every speaker article/card has the exact **width**, **height**, **color**, **font** and **size** as defined on the original design. Please fix this in order to make your design stick to the original one as much as possible :+1: 

#### Original design
![Screenshot from 2022-12-02 21-05-55](https://user-images.githubusercontent.com/6615394/205388177-2628080d-1bfd-4ea5-bf5e-caf0674a7125.jpg)